commands/snap.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Canonical Ltd.
+// Copyright 2015 Cloudbase Solutions SRL
```

Comment:
Is this license needed?

Reply:
I'm not sure. It's included in the apt file and I've copied it straight from there. So I've assumed that there are still elements of the original copyright that have been preserved.